protected/views/site/login.php

```diff
@@ -1,69 +1,79 @@
-        <div class="col-xs-12">
-            <?php if(Yii::app()->user->hasFlash('success-reset-password')): ?>
-                <div class="alert alert-success">
-                    <?php echo Yii::app()->user->getFlash('success-reset-password'); ?>
-                </div>
-            <?php endif; ?>
-        </div>
+<section>
+    <div class="container" id="login">
+        <section class="page-title-section">
+            <div class="page-title">
+                <ol class="breadcrumb pull-right">
+                    <li><a href="/">Home</a></li>
+                    <li class="active">login</li>
+                </ol>
+                <h1 class="h4">Login</h1>
+            </div>
+        </section>
+        <div class="subsection row" style="margin-bottom: 130px;">
+            <div class="col-xs-12">
+                <?php if (Yii::app()->user->hasFlash('success-reset-password')) : ?>
+                    <div class="alert alert-success">
+                        <?php echo Yii::app()->user->getFlash('success-reset-password'); ?>
+                    </div>
+                <?php endif; ?>
+            </div>
             <div class="col-xs-6">
                 <div class="subsection-login">
-                    <p><?=Yii::t('app' , 'Please fill out the following form with your login credentials:')?></p>
-                    <p><?=Yii::t('app' , 'Fields with <span class="symbol">*</span> are required.')?></p>
+                    <p><?= Yii::t('app', 'Please fill out the following form with your login credentials:') ?></p>
+                    <!-- inputs are already announced as required so screen readers do not need this message -->
+                    <p aria-hidden="true"><?= Yii::t('app', 'Fields with <span class="symbol">*</span> are required.') ?></p>
                 </div>
-               
-                <div class="login-div">
-			        <? $form = $this->beginWidget('CActiveForm', array('htmlOptions'=>array('class'=>'form-horizontal'))) ?>
-			    <div class="form-group">
-				<?= $form->labelEx($model,'username', array('class'=>'col-xs-3 control-label')) ?>
-				<div class="col-xs-9">
-					<?= $form->textField($model,'username',array('size'=>50,'class'=>'form-control')) ?>
-					<?php echo $form->error($model,'username'); ?>
-				</div>
 
-			</div>
+                <div class="login-div">
+                    <? $form = $this->beginWidget('CActiveForm', array('htmlOptions' => array('class' => 'form-horizontal'))) ?>
+                    <div class="form-group">
+                        <label class="col-xs-3 control-label error required" for="LoginForm_username">
+                            Email Address<span class="required" aria-hidden="true">*</span>
+                        </label>
+                        <div class="col-xs-9">
+                            <?= $form->textField($model, 'username', array('size' => 50, 'class' => 'form-control', 'aria-describedby' => 'usernameError', 'required' => true)) ?>
```

Comment:
Added `aria-describedby` to connect the input and its error, and `required` for client side validation